packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -125,9 +129,14 @@ export class DocumentLoad extends BasePlugin<unknown> {
     entries: PerformanceEntries
   ) {
     // span can be undefined when entries are missing the certain performance - the span will not be created
-    if (typeof span !== 'undefined' && hasKey(entries, performanceName)) {
-      this._addSpanEvent(span, performanceName, entries);
-      span.end(entries[performanceName]);
+    if (typeof span !== 'undefined') {
```

Comment:
If span is undefined would `if (!span)` throw an error?

Reply:
nope